filter_plugins/filter_snake_case.py

```diff
@@ -1,10 +1,14 @@
-# copied from https://github.com/infrawatch/tripleo-qdr-ansible-role/commit/4897cb6a09fd83c12b8d7a578c845471afe68dbc
+# copied from
```

Comment:
BTW as part of my review I've updated this again slightly and moved it into it's correct home (https://github.com/openstack/tripleo-ansible/tree/master/tripleo_ansible/ansible_plugins/filter). It hasn't been posted yet, but that will be the eventual fate of this filter.

Reply:
Awesome, I'll make this file go away when I move to the tripleo_ansible repo!